src/server/req_stat.c

```diff
@@ -478,28 +476,23 @@ req_stat_que(struct batch_request *preq)
 			}
 		}
 		if (consider_q) {
-			rc = status_que(pque, preq, &preply->brp_un.brp_status,
-					(preq->rq_extend ? &attr : NULL));
+			rc = status_que(pque, preq, &preply->brp_un.brp_status);
 			consider_q = 0;
 		}
 	} else {	/* get status of queues */
-		pque = (pbs_queue *)GET_NEXT(svr_queues);
-		while (pque) {
+		for (pque = (pbs_queue *)GET_NEXT(svr_queues); pque;pque = (pbs_queue *)GET_NEXT(pque->qu_link)) {
```

Comment:
Add a space after the ';'

Reply:
This is outdated in the sense that we are now doing the filtering of queues at the scheduler side.